metadata/ComputerVision__9__metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "changeLog": "Release v2022.10.10",
+  "cpu": 0,
+  "description": "Backend server for the Computer Vision solution that detects User Interface Elements from a provided Application screenshot",
+  "displayName": "ComputerVision",
+  "gpu": 1,
+  "inputDescription": "Please use the Computer Vision Activities included in the UIAutomation package. For more information please visit the official documentation.",
```

Comment:
Optional: Please include a link to the official documentation, you can see the messages used in the latest metadata files (for 23.10 for example).

Reply:
Done.